<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zalogowany</class>
 <widget class="QMainWindow" name="Zalogowany">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snooker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="opcjeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Administratorzy</string>
     </attribute>
     <widget class="QPushButton" name="dodajAdministratora">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj Administratora</string>
      </property>
     </widget>
     <widget class="QFrame" name="administratorFrame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>351</width>
        <height>231</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="imieText">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nazwiskoText">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="loginText">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="hasloText">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="imieLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Imię</string>
       </property>
      </widget>
      <widget class="QLabel" name="nazwiskoLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Nazwisko</string>
       </property>
      </widget>
      <widget class="QLabel" name="loginLabel">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLabel" name="hasloLabel">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>90</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Hasło</string>
       </property>
      </widget>
      <widget class="QPushButton" name="edycjaAdministratora">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Dodaj</string>
       </property>
      </widget>
      <widget class="QLabel" name="errorLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QListWidget" name="administratorzyListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Zawodnicy</string>
     </attribute>
     <widget class="QListWidget" name="zawodnicyListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj Zawodnika</string>
      </property>
     </widget>
     <widget class="QFrame" name="zawodnikFrame">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>361</width>
        <height>231</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="imieZawodnikLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Imię</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="imieZawodnikLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nazwiskoZawodnikLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="nazwiskoZawodnikLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Nazwisko</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="narodowoscZawodnikLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="narodowoscZawodnikLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Narodowość</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dataUrodzeniaDateEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximumDateTime">
        <datetime>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
         <year>2100</year>
         <month>12</month>
         <day>31</day>
        </datetime>
       </property>
       <property name="minimumDateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>1900</year>
         <month>9</month>
         <day>14</day>
        </datetime>
       </property>
      </widget>
      <widget class="QLabel" name="dataUrodzeniaZawodnikLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Data Urodzenia</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="zarobkiZawodnikLineEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="zarobkiZawodnikLabel">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>10</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Zarobki</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="najwyzszyBrejkZawodnikLineEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="najwyzszyBrejkZawodnikLabel">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Najwyższy Brejk</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="brejkiStupunktoweZawodnikLineEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="brejkiStupunktoweZawodnikLabel">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Brejki Stupunktowe</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="brejkiMaksymalneZawodnikLineEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="brejkiMaksymalneZawodnikLabel">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Brejki Maksymalne</string>
       </property>
      </widget>
      <widget class="QPushButton" name="edytujZawodnikaButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>200</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Dodaj</string>
       </property>
      </widget>
      <widget class="QLabel" name="errorZawodnikLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Turnieje</string>
     </attribute>
     <widget class="QListWidget" name="turniejeListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>171</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="zawodnicyTurniejListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>291</y>
        <width>171</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="meczeListWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>171</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="partieListWidget">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>171</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="turniejeLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Turnieje</string>
      </property>
     </widget>
     <widget class="QLabel" name="zawodnicyTurniejLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Zawodnicy</string>
      </property>
     </widget>
     <widget class="QLabel" name="meczeLabel">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Mecze</string>
      </property>
     </widget>
     <widget class="QLabel" name="partieLabel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Partie</string>
      </property>
     </widget>
     <widget class="QFrame" name="turniejFrame">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>240</y>
        <width>371</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="nazwaTurniejLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nazwa</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nazwaTurniejLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="sponsorTurniejLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Sponsor</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sponsorTurniejLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="miejsceTurniejLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Miejsce</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="miejsceTurniejLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="pulaNagrodTurniejLabel">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pula Nagród (PLN)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="pulaNagrodTurniejLineEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="najwyzszyBrejkTurniejLabel">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Najwyższy Brejk</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="najwyzszyBrejkTurniejLineEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="liczbaZawodnikowTurniejLabel">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Liczba Zawodników</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="liczbaZawodnikowTurniejLineEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="edytujTurniejButton">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>160</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Dodaj</string>
       </property>
      </widget>
      <widget class="QListWidget" name="dodajZawodnikaListWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="dodajZawodnikaPushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Dodaj Do Turnieju</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="dodajTurniejButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj Turniej</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
